index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;600&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <div class="menu-bar">
+      <div class="menu">
+        <div id="bar1" class="bar"></div>
+        <div id="bar2" class="bar"></div>
+        <div id="bar3" class="bar"></div>
```

Comment:
- Kindly make sure your navigation bar design looks like the Figma design as required by the previous reviewer. follow this  [link](https://github.com/akhror-valiev/micronauts-portfolio/pull/1/files/6ca7b69ff39513d968726a40c01ba92380660e37#r977240187) to see changes suggest by the first reviewer :grinning: : :grimacing: